### PR TITLE
Integrate into PowerToys.sln

### PR DESCRIPTION
**Pull request by stefansjfw**
## Summary of the Pull Request
 - integrate all projects into PowerToys.sln
 - remove all other .sln files
 - remove unneeded files IMO (LET ME KNOW IF SOME OF THOES FILES ARE ACTUALLY NEEDED!)

## PR Checklist

- [ ] **Closes:** #xxx
- [ ] **Communication:** I've discussed this with core contributors already. If work hasn't been agreed, this work might be rejected
- [ ] **Tests:** Added/updated and all pass
- [ ] **Localization:** All end user facing strings can be localized
- [ ] **Dev docs:** Added/updated
- [ ] **New binaries:** Added on the required places
   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
   - [ ] [YML for CI pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects
   - [ ] [YML for signed pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/release.yml)
- [ ] **Documentation updated:** If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) and link it here: #xxx

### CODE REVIEW

File: src/modules/cmdpal/.wt.json [hunk 1]
```diff
@@ -1,16 +0,0 @@
-{
```
Comment (zadjii-msft) [inline src/modules/cmdpal/.wt.json:1]: Oh this one's actually a new-ish Terminal feature - you can stick commandline snippets in a directory to have them displayed in the Terminal. This one we should keep ☺️

File: src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Microsoft.CmdPal.UI.ViewModels.csproj [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```
Comment (zadjii-msft) [inline src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Microsoft.CmdPal.UI.ViewModels.csproj:6]: this one didn't need `<OutputPath>`?

File: src/modules/cmdpal/extensionsdk/nuget/Microsoft.Windows.CommandPalette.Extensions.nuspec [hunk 3]
```diff
@@ -1,36 +0,0 @@
-<?xml version="1.0"?>
```
Comment (zadjii-msft) [inline src/modules/cmdpal/extensionsdk/nuget/Microsoft.Windows.CommandPalette.Extensions.nuspec:1]: (More stuff that I think @ericjohnson327 will probably end up resurrecting soon-ish)

File: src/modules/cmdpal/extensionsdk/BuildSDKHelper.ps1 [hunk 4]
```diff
@@ -1,102 +0,0 @@
-Param(
```
Comment (zadjii-msft) [inline src/modules/cmdpal/extensionsdk/BuildSDKHelper.ps1:1]: @ericjohnson327 I suspect you'll end up bringing this back soon<sup>TM</sup>, but I think it's fine to prune this currently definitely unused version out. I'll let you commit a real version in the future ☺️

**Review by zadjii-msft — COMMENTED** [review]
Couple minor comments but nothing blocking. I'll probably merge it like tomorrow

File: src/modules/cmdpal/.wt.json (hunk 1 above)
Comment (stefansjfw) [reply src/modules/cmdpal/.wt.json:1]: oh, ok.. didn't know that :) Reverting

File: src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Microsoft.CmdPal.UI.ViewModels.csproj (hunk 2 above)
Comment (stefansjfw) [reply src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Microsoft.CmdPal.UI.ViewModels.csproj:6]: not necessarily, it ends up where needed as a dependency, but let's add it to align with others.

**Review by zadjii-msft — APPROVED** [review]

**Pull request merged**